packages/@react-types/toast/src/index.d.ts

```diff
@@ -13,22 +13,30 @@
 import {DOMProps, StyleProps} from '@react-types/shared';
 import {ReactNode} from 'react';
 
-export interface ToastOptions {
+export interface ToastOptions extends DOMProps, StyleProps {
```

Comment:
why do the options have dom and style props?

Reply:
StyleProps is included for this line of code https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-spectrum/toast/src/Toast.tsx#L49

DOMProps is included for this line of code https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-aria/toast/src/useToast.ts#L34